### PR TITLE
phpstan lvl 1

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: inc/container.class.php [hunk 1]
```diff
@@ -487,7 +489,7 @@ public function getValueToSelect($field_id_or_search_options, $name = '', $value
                 return Dropdown::showFromArray($name, self::getTypes(), $options);
             case $this->getTable() . '.itemtypes':
                 $options['display'] = false;
-                return Dropdown::showFromArray($name, self::getItemtypes(), $options);
+                return Dropdown::showFromArray($name, self::getItemtypes(false), $options);
```
Comment (trasher) [inline inc/container.class.php:492]: I'm not really sure about this one; I do not know if there is a way here to find what to pass

File: phpstan.neon [hunk 2]
```diff
@@ -0,0 +1,17 @@
+parameters:
+    parallel:
+        maximumNumberOfProcesses: 2
+    level: 1
+    bootstrapFiles:
+        - ../../inc/based_config.php
+        - setup.php
```
Comment (cedric-anne) [inline phpstan.neon:7]: Shopuld be moved in `paths` list, so errors on this file would be detected.

File: phpstan.neon [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    paths:
+        - inc
+        - front
+        - ajax
+        - hook.php
+    scanDirectories:
+        - ../../inc
+        - ../../src
+    stubFiles:
+        - ../../stubs/glpi_constants.php
```
Comment (cedric-anne) [inline phpstan.neon:17]: Requires an upgrade of the `glpi-project/tools` lib.

```suggestion
        - ../../stubs/glpi_constants.php
rules:
    - GlpiProject\Tools\PHPStan\Rules\GlobalVarTypeRule
```

File: inc/container.class.php (hunk 1 above)
Comment (cedric-anne) [reply inc/container.class.php:492]: It seems OK.

**Pull request merged**